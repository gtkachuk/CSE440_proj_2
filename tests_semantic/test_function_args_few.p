program testFunctionCallObject;


class testFunctionCallObject

BEGIN
   
   VAR
      retval  : integer;
      counter : integer;

FUNCTION test(value1 : integer; value2: integer): integer;
BEGIN
   counter := value2;
   test := 3
END;

FUNCTION testFunctionCallObject;
BEGIN
   retval := test(4)
END

END
.
