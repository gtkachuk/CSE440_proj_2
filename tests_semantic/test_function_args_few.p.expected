ERROR:21:Function "test" not declared
Errors detected. Exiting.
